Xamarin.Forms.Controls/RootPages/RootFluentPage.cs

```diff
@@ -0,0 +1,227 @@
+﻿using Xamarin.Forms.Fluent;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Collections.Concurrent;
+using System.Windows.Input;
+using System;
+using System.Collections.Generic;
+using static System.Math;
+
+namespace Xamarin.Forms.Controls.RootPages
+{
+	public class RootFluentPage : ContentPage
+	{
+		public RootFluentPage()
+			=> Content = new StackLayout
+			{
+				Padding = new Thickness(25, 50),
+				Children =
+					{
+						new Label()
+							.Bind(Label.TextProperty, nameof(RootFluentViewModel.LabelText))
+							.Bind(BackgroundColorProperty, nameof(RootFluentViewModel.LabelBackgroundColor),
+								converter: new GenericValueConverter(v => Color.FromHex(v.ToString())))
+							.Bind(HeightRequestProperty, nameof(RootFluentViewModel.LabelHeight)),
+
+						new Button
+						{
+							Text = "Click Me (1)",
+							CommandParameter = "(1) Clicked"
+						}.Command(nameof(RootFluentViewModel.ClickCommand)),
+
+						new Button
+						{
+							Text = "Click Me (2)",
+							CommandParameter = "(2) Clicked"
+						}.Command(nameof(RootFluentViewModel.ClickCommand)),
+
+						new ContentView
+						{
+							BackgroundColor = Color.Gold,
+							Content = new Label
+							{
+								FontSize = 20,
+								Text = "Click Me (3)"
+							}
+						}.Tap(nameof(RootFluentViewModel.ClickCommand), "(3) Clicked")
+					}
+			}.Fluent(p => BindingContext = new RootFluentViewModel());
```

Comment:
why don't you set the context like any other property ?

Reply:
@StephaneDelcroix i did it for demonstrating all current possibilities of fluent layout.

it can be useful for subscribing on events etc.

``` csharp
...
.Fluent(view => view.PropertyChanged += (s, e) => Debug.WriteLine(e.PropertyName)); ```